crates/biome_cli/src/diagnostics.rs

```diff
@@ -262,6 +264,12 @@ impl DeprecatedArgument {
     }
 }
 
+#[derive(Debug, Diagnostic)]
+pub enum ReportDiagnostic {
+    /// Emitted when trying to serialise the report
+    Serialisation(SerdeJsonError),
```

Comment:
```suggestion
    Serialization(SerdeJsonError),
```

Writing Serialisation with an "s" is UK English. I prefer UK English myself (I use it to write my novels!), but for programming-related matters, US English is the gold standard.